server/handlers/meshery_pattern_handler.go

```diff
@@ -1259,27 +1245,10 @@ func (h *Handler) DownloadMesheryPatternHandler(
 		pattern.PatternFile = patternFileStr
 	}
 
-	if formatConverter != nil {
-		patternFile, err := formatConverter.Convert(pattern.PatternFile)
-		if err != nil {
-			err = ErrExportPatternInFormat(err, exportFormat, pattern.Name)
-			h.log.Error(err)
-			http.Error(rw, err.Error(), http.StatusInternalServerError)
-			return
-		}
-		rw.Header().Add("Content-Disposition", fmt.Sprintf("attachment;filename=%s.yml", pattern.Name))
-		rw.Header().Set("Content-Type", "application/yaml")
-		_, err = fmt.Fprint(rw, patternFile)
-		if err != nil {
-			err = ErrWriteResponse(err)
-			h.log.Error(err)
-			http.Error(rw, _errors.Wrapf(err, "failed to export design \"%s\" in %s format", pattern.Name, exportFormat).Error(), http.StatusInternalServerError)
-			return
-		}
-		return
-	}
+	exportFormat := r.URL.Query().Get("export")
 
-	if ociFormat {
+	// for downloading oci image
+	if (exportFormat=="OCI Image") {
```

Comment:
Thanks for fixing the issue, however, I would suggest that `OCI Image` text should be `oci` only, current one has text case sensitivity issues as we are not sanitizing it to a single format also there is no need for space, "oci" should be enough.

Are we sending "OCI Image" from frontend? @amitamrutiya can you confirm is this same case inside the Kanvas?

Reply:
@Yashsharma1911 We are sending "OCI Image" from the UI in the import model. It's the same case in Kansas, too, since it uses the same model.